tests/unit_tests/test_data_multipole.py

```diff
@@ -46,3 +46,19 @@ def test_export_to_hdf5(tmpdir, u235):
     filename = str(tmpdir.join('092235.h5'))
     u235.export_to_hdf5(filename)
     assert os.path.exists(filename)
+
+
+def test_from_endf():
+    pytest.importorskip('vectfit')
+    endf_data = os.environ['OPENMC_ENDF_DATA']
+    endf_file = os.path.join(endf_data, 'neutrons', 'n-001_H_001.endf')
+    return openmc.data.WindowedMultipole.from_endf(
+            endf_file, wmp_options={"n_win": 400, "n_cf": 3})
```

Comment:
Now that search is turned on by default, we need to disable it for this test:
```suggestion
            endf_file, wmp_options={"search": False, "n_win": 400, "n_cf": 3})
```

Reply:
My bad. `{"n_win": 400, "n_cf": 3}` somehow indicates we don't want `search`, so I added a check in the code to set `search=False` automatically in this case.